app/src/controllers/administrate/sagas.js

```diff
@@ -37,10 +51,52 @@ function* fetchProjectData() {
   yield put(sectionDataAction(projectKey, isAdminAccess));
 }
 
+function* fetchActiveProjectData({ payload: { organizationSlug, projectSlug, projectKey } }) {
+  try {
+    yield put(fetchOrganizationBySlugAction(organizationSlug));
+
+    if (projectKey) {
+      yield put(setActiveProjectKeyAction(projectKey));
+      yield put(fetchProjectAction(projectKey));
+    } else {
+      yield take(createFetchPredicate(FETCH_ORGANIZATION_BY_SLUG));
+      yield put(fetchOrganizationProjectsAction(organizationSlug));
+      yield take(createFetchPredicate(FETCH_ORGANIZATION_PROJECTS));
+
+      const organizationProjects = yield select(organizationProjectsSelector);
```

Comment:
Please leave a TODO comment to rewrite to it to 'fetch org project by slug' in the future.

Reply:
Done.